community/www/batch/become-a-member/index.html

```diff
@@ -0,0 +1,73 @@
+% extends "templates/base.html" %}
+{% block title %}Become a member{% endblock %}
+{% block head_include %}
+<meta name="description" content="Become a member" />
+<meta name="keywords" content="" />
+{% endblock %}
+
+{% block content %}
+{% if frappe.session.user == "Guest" %}
+
+<div class="page-card">
+  <div class='page-card-head'>
+    <span class='indicator blue password-box'>Login Required</span>
+  </div>
+  <div class=''>Please log in to confirm your membership.</div>
+  <a type="submit" id="login" class="btn btn-primary w-100"
+    href="/login?redirect-to=/become-a-member/{{ batch.name }}">{{_("Login")}}</a>
+</div>
+
+{% elif already_a_member %}
+
+<div class="page-card">
+  <div class='page-card-head'>
+    <span class='indicator blue password-box'>Already a member</span>
+  </div>
+  <div class=''>You are already a member of the batch {{ batch.title }}.
```

Comment:
Again, it is not clear which course it is. I think it is important to mention both the course title and the batch name.